projects/js-packages/components/components/admin-page/index.jsx

```diff
@@ -24,17 +25,21 @@ import Col from '../layout/col';
  * @returns {React.Component} AdminPage component.
  */
 const AdminPage = props => {
-	const { children, moduleName, a8cLogoHref, showHeader, showFooter, showBackground } = props;
+	const { children, moduleName, a8cLogoHref, showHeader, showFooter, showBackground, logo } = props;
+
 	const rootClassName = classNames( styles[ 'admin-page' ], {
 		[ styles.background ]: showBackground,
 	} );
 
+	// Filter valid logo properties.
+	const logoProps = pick( logo, [ 'className', 'height', 'showText', 'logoColor' ] );
```

Comment:
Why do not pass everything, this will avoid us to maintain this list and in some moment forget some new added prop on `logo`.

Reply:
I prefer do not to pass properties that the component doesn't expect. You're right about the maintenance part, though.

Reply:
removed